crmprtd/process.py

```diff
@@ -109,6 +111,22 @@ def get_normalization_module(network):
     return import_module(f"crmprtd.networks.{network}.normalize")
 
 
+@functools.lru_cache(maxsize=None)
+def get_network(sesh, var_id):
```

Comment:
The `sesh` variable isn't cacheable since it is an object of a database session which necessarily involves "state". I suspect that you were trying to do something similar to the `cached_function` decorator that you imported above (but don't actually use)?

If you choose to use `cached_function` here, I think that you should move it out of the `align` module to another module that is higher level and would be intended to be used across the entire code base.